### ISSUE TITLE
I cannot decode this image.

### ISSUE THREAD

**Issue by ITsolution-git** [cozmo/jsQR#54]
![screenshot from 2018-03-25 13 57 21](https://user-images.githubusercontent.com/24530559/37931020-5e0a1564-3112-11e8-9951-bb39243be4e7.png)
![screenshot from 2018-03-26 16 19 11](https://user-images.githubusercontent.com/24530559/37931022-5e4bbdc0-3112-11e8-95b6-ea2bcdbe7d20.png)
Would you please help me?
The jsQr gives me error with these images?
Especially on colored qr codes.

Thank you in advance.

**Comment by cozmo** [comment]
Both these images seem to be related to the contrast ratio - the first one the contrast ratio is low and the second is being thrown off because the traditionally black squares are white - it's basically inverted. 

Both of these issues should be able to be solved - I added test cases here - 9c8a6f0e3aba2f05a002c73325cbc5d89062df73 and so we'll start to see the tests pass as we improve the detection with relation to contrast ratios.

**Comment by ITsolution-git** [comment]
Awesome. Thank you @cozmo.

**Comment by cozmo** [comment]
Now that #67 has been merged we're...closer. Still failing to binarize your images though I think, which is odd. Will let you know when this issue is resolved.

**Comment by ITsolution-git** [comment]
Thank you.
